org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/XMLCompletions.java

```diff
@@ -125,6 +126,17 @@ public CompletionList doComplete(DOMDocument xmlDocument, Position position, Sha
 						return completionResponse;
 					}
 					break;
+				case DTDStartDoctypeTag:
+					DTDDeclParameter systemId = xmlDocument.getDoctype().getSystemIdNode();
+					if (systemId == null) {
+						break;
+					}
+					if (systemId.getStart() <= offset && offset <= systemId.getEnd()) {
+						collectAttributeValueSuggestions(systemId.getStart(), systemId.getEnd(),

```

Comment:
Although the systemId parameter is not an attribute, I decided to call `collectAttributeValueSuggestions` since file path completion for attribute values were already implemented in `FilePathCompletionParticipant#onAttributeValue()`.

Reply:
@angelozerr @datho7561 actually, maybe it might be better to move the contents of https://github.com/eclipse/lemminx/blob/59c04ef37c7959d95044f61564939c2f139b2d06/org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/general/completion/FilePathCompletionParticipant.java#L48

into a new static method and have:
```
case DTDStartDoctypeTag:
...
...
call static method
```
instead of
```
case DTDStartDoctypeTag:
...
...
collectAttributeValueSuggestions()
```

Reply:
I will make a new commit for this

Reply:
You mean that XMLCompletion (core) will depends on FilePathCompletionParticipant(extension). I would liek to avoid doing that.

Reply:
I think the clean mean is to have a onDTDSystemId

Reply:
Ok thank you, I created a new commit that creates onDTDSystemId. If it looks good, I will squash my two commits into one